tests/test_logging.py

```diff
@@ -1,11 +1,7 @@
-import pytest
-
-
-@pytest.mark.usefixtures("mock_get_response_xlsx")
-def test_download_logging(flask_test_client, caplog):
+def test_download_logging(flask_test_client, caplog, mocked_process_async_download):
     flask_test_client.post("/download", data={"file_format": "xlsx"})
```

Comment:
Can we pass an email address through here and see it gets passed in?

Reply:
Added email to mocked_auth function in tests/conftest.py, now the user's email address goes through and gets logged.